xsnap/makefiles/mac/xsnap-worker.mk

```diff
@@ -72,8 +62,19 @@ ifneq ("x$(SDKROOT)", "x")
 	LINK_OPTIONS += -isysroot $(SDKROOT)
 endif
 
-# C_OPTIONS += -fsanitize=address -fno-omit-frame-pointer
-# LINK_OPTIONS += -fsanitize=address -fno-omit-frame-pointer
+## present in 3.9.2 xst.mk
+## C_OPTIONS += -fsanitize=address -fno-omit-frame-pointer
+## LINK_OPTIONS += -fsanitize=address -fno-omit-frame-pointer
+# present in 5.5.0 xst.mk
+#	ifeq ($(SANITIZER), undefined)
+#		C_OPTIONS += -fsanitize=bool,builtin,enum,integer-divide-by-zero,null,object-size,return,returns-nonnull-attribute,shift,signed-integer-overflow,unreachable,vla-bound,vptr -fno-sanitize-recover=bool,builtin,enum,integer-divide-by-zero,null,object-size,return,returns-nonnull-attribute,shift,signed-integer-overflow,unreachable,vla-bound,vptr,array-bounds,function
+#		LINK_OPTIONS += -fsanitize=bool,builtin,enum,integer-divide-by-zero,null,object-size,return,returns-nonnull-attribute,shift,signed-integer-overflow,unreachable,vla-bound,vptr -fno-sanitize-recover=bool,builtin,enum,integer-divide-by-zero,null,object-size,return,returns-nonnull-attribute,shift,signed-integer-overflow,unreachable,vla-bound,vptr,array-bounds,function
+#	else
+#		C_OPTIONS += -fsanitize=address -fsanitize-blacklist=xst_no_asan.txt
+#		LINK_OPTIONS += -fsanitize=address
+#	endif
+#	C_OPTIONS += -DmxASANStackMargin=131072 -fno-omit-frame-pointer
+#	LINK_OPTIONS += -fno-omit-frame-pointer
```

Comment:
Please confirm I can safely delete this commented section that I found in xst.mk on Moddable 5.5.0.

Reply:
That all appears to be related to fuzzer configurations. You should be able to report that.

FWIW – it might be interesting to run xsnap with sanitizers enabled in a debug / development environment. It certainly uses XS in ways that are different from the fuzzers and would also check your name code. That should be an option since the performance impact of the sanitizers is considerable.